tuxemon/event/actions/play_map_animation.py

```diff
@@ -77,8 +77,7 @@ def start(self) -> None:
         elif self.parameters.loop == "noloop":
             loop = False
         else:
-            logger.error('animation loop value must be "loop" or "noloop"')
-            raise ValueError
+            raise ValueError('animation loop value must be "loop" or "noloop"')
```

Comment:
Sorry, I didn't read your message properly, and assumed that the one you were fixing was within one of the files you were already editing.

I'd rather not have out scope changes applied to random files, please undo this and all the others.

Given that it's a common pattern in this corebase I don't mind which way you throw the exception for your new code, go for whatever is easiest.

Reply:
If possible keep the change somewhere in case we want to reapply the change in another PR.

Reply:
Sure, I got carried away, I'll open another PR.